sale_automatic_workflow_payment_mode/models/automatic_workflow_job.py

```diff
@@ -21,7 +21,7 @@ def _prepare_dict_account_payment(self, invoice):
         return vals
 
     def _register_payment_invoice(self, invoice):
-        if not invoice.payment_mode_id.fixed_journal_id:
+        if invoice.payment_mode_id and not invoice.payment_mode_id.fixed_journal_id:
```

Comment:
@SodexisTeam Apart this is correct, note that if payment_mode_id is False, ORM will compute result of invoice.payment_mode_id.fixed_journal_id as False too.

So, this change is not necessary

Reply:
@rousseldenis please review my last comment